client/src/components/Settings.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function Settings({ fuelValue, setFuelValue }) {
+  return (
+    <Wrapper>
+      <div>
```

Comment:
Do you need this div?

Reply:
We do not need, thank you also at this point for your hint.